content/dart/concepts/variables/variables.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Variables'
+Description: 'A variable is a location in computer memory used to store data.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Variables'
+  - 'Data Types'
+---
+
+A **`variable`** is a location in computer memory used to store references, usually for use in a program. Variables serve as symbolic names (identifiers) for values in the computer's memory. Variables allow programmers to manipulate and process data dynamically within their programs.
+
+## Syntax
+
+When declaring a variable in Dart, the type of variable goes first, followed by the name, and then the value:
+
+```pseudo
+type name = 'value';
+```
+
+The default type 'var' is inferred to be a `string`. If an object isn't restricted to a single type, specify the Object type (or dynamic if necessary):
```

Comment:
Since we are adding all the values of `type` can take above, let's make this a note. 
``` > **Note:** text inside note ```